psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -1083,6 +1083,26 @@ void HF::guess() {
         sad_ = true;
         guess_E = compute_initial_E();
 
+    } else if (guess_type == "HUCKEL") {
+        if (print_) outfile->Printf("  SCF Guess: Huckel guess via on-the-fly atomic UHF (doi:10.1021/acs.jctc.8b01089).\n\n");
+
+        // Huckel guess, written by Susi Lehtola 2019-01-27.  See "An
+        // assessment of initial guesses for self-consistent field
+        // calculations. Superposition of Atomic Potentials: simple
+        // yet efficient", JCTC 2019, doi: 10.1021/acs.jctc.8b01089.
```

Comment:
can add to https://github.com/psi4/psi4/blob/master/doc/sphinxman/source/introduction.rst#citing-psifour (may need to add a SCF section)

Reply:
Done. Doesn't really make sense to hide it in a SCF section, since a guess is needed regardless of the type of calculation.